alonzo/test/lib/Test/Cardano/Ledger/Alonzo/Examples/Consensus.hs

```diff
@@ -0,0 +1,174 @@
+import Cardano.Ledger.Coin (Coin (..))
+import Cardano.Ledger.Core (TxBody)
+import Cardano.Ledger.Crypto (Crypto, StandardCrypto)
+import Cardano.Ledger.Era (ValidateScript (hashScript))
+import Cardano.Ledger.Keys (asWitness)
+import Cardano.Ledger.Mary.Value (AssetName (..), PolicyID (..), Value (..))
+import Cardano.Ledger.SafeHash (hashAnnotated)
+import Cardano.Ledger.ShelleyMA.Timelocks (Timelock (..), ValidityInterval (..))
+import Cardano.Slotting.Slot (EpochNo (..), SlotNo (..))
+import Data.Default.Class (def)
+import qualified Data.Map.Strict as Map
+import Data.Proxy (Proxy (..))
+import qualified Data.Sequence.Strict as StrictSeq
+import qualified Data.Set as Set
+import qualified PlutusTx as Plutus
+import Shelley.Spec.Ledger.API
+  ( ApplyTxError (..),
+    Credential (..),
+    Network (..),
+    NewEpochState (..),
+    ProposedPPUpdates (..),
+    RewardAcnt (..),
+    TxId (..),
+    TxIn (..),
+    Update (..),
+    Wdrl (..),
+  )
+import Shelley.Spec.Ledger.STS.Delegs (DelegsPredicateFailure (..))
+import Shelley.Spec.Ledger.STS.Ledger (LedgerPredicateFailure (..))
+import Shelley.Spec.Ledger.Tx (Tx (..))
+import Shelley.Spec.Ledger.UTxO (makeWitnessesVKey)
+import qualified Test.Shelley.Spec.Ledger.Examples.Consensus as SLE
+import Test.Shelley.Spec.Ledger.Orphans ()
+import Test.Shelley.Spec.Ledger.Utils (mkAddr)
+
+type StandardAlonzo = AlonzoEra StandardCrypto
+
+-- | ShelleyLedgerExamples for Alonzo era
+ledgerExamplesAlonzo :: SLE.ShelleyLedgerExamples StandardAlonzo
+ledgerExamplesAlonzo =
+  SLE.ShelleyLedgerExamples
+    { SLE.sleBlock = SLE.exampleShelleyLedgerBlock exampleTransactionInBlock,
+      SLE.sleHashHeader = SLE.exampleHashHeader (Proxy @StandardAlonzo),
+      SLE.sleTx = exampleTx,
+      SLE.sleApplyTxError =
+        ApplyTxError $
+          pure $
+            DelegsFailure $
+              DelegateeNotRegisteredDELEG @StandardAlonzo (SLE.mkKeyHash 1),
+      SLE.sleRewardsCredentials =
+        Set.fromList
+          [ Left (Coin 100),
+            Right (ScriptHashObj (SLE.mkScriptHash 1)),
+            Right (KeyHashObj (SLE.mkKeyHash 2))
+          ],
+      SLE.sleResultExamples = resultExamples,
+      SLE.sleNewEpochState = exampleAlonzoNewEpochState,
+      SLE.sleChainDepState = SLE.exampleLedgerChainDepState 1
+    }
+  where
+    resultExamples =
+      SLE.ShelleyResultExamples
+        { SLE.srePParams = def,
+          SLE.sreProposedPPUpdates = examplePPPU,
+          SLE.srePoolDistr = SLE.examplePoolDistr,
+          SLE.sreNonMyopicRewards = SLE.exampleNonMyopicRewards,
+          SLE.sreShelleyGenesis = SLE.testShelleyGenesis
+        }
+    examplePPPU =
+      ProposedPPUpdates $
+        Map.singleton
+          (SLE.mkKeyHash 0)
+          (emptyPParamsUpdate {_collateralPercentage = SJust 150})
+
+exampleMultiAssetValue ::
+  forall c.
+  Cardano.Ledger.Crypto.Crypto c =>
+  Int ->
+  Cardano.Ledger.Mary.Value.Value c
+exampleMultiAssetValue x =
```

Comment:
Maybe this could be unified with [this](https://github.com/input-output-hk/cardano-ledger-specs/blob/master/shelley-ma/shelley-ma-test/src/Test/Cardano/Ledger/Mary/Examples/Consensus.hs#L31) and there use `exampleMultiAssetValue 1`, as this is somewhat "mary related" instead of specific to Alonzo.

Reply:
yep, great idea, thanks!

Reply:
done